Toolchain/BuildPython.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+
+# This file will need to be run in bash, for now.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+echo "$DIR"
+
+TARGET=i686-pc-serenity
+PREFIX="$DIR/Local"
+SYSROOT="$DIR/../Root"
+
+source ../Ports/python-3.6/version.sh
```

Comment:
This should be relative to `$DIR` instead of assuming what cwd is.

Reply:
Good point! I'll fix it.